conda_build_prepare/git_helpers.py

```diff
@@ -145,10 +145,11 @@ def _call_custom_git_cmd(git_repo, cmd_string, check=True, quiet=False):
     return stdout.strip()
 
 
-def get_latest_describe_tag(git_repo):
-    # Find the `git describe` tag having any version-like part
+def get_latest_tag(git_repo):
```

Comment:
I think I'm struggling with the `latest` in the `get_latest_tag` here. I can't really think of a better word. I don't really like `get_top_tag` either.

I think the code is getting any tag which happens to be in the history for the current commit?

Reply:
yep, maybe `first_reachable` is a better phrasing?